src/main.cpp

```
@@ -115,127 +100,96 @@ RE::NiSourceTexturePtr* GetTextureFromIndex(RE::BSLightingShaderMaterial* materi
+	if (node->extraDataSize > 0) {
+		for (uint16_t i = 0; i < node->extraDataSize; i++) {
+			logger::info(
 				"{} {}",
 				node->extra[i]->GetRTTI()->name,
-				node->extra[i]->name
-			);
+				node->extra[i]->name);
 		}
 	}
 
 	RE::NiNode* niNode = node->AsNode();
```

Clang (14.0.6)

[warning] variable 'niNode' is not initialized

(cppcoreguidelines-init-variables)